README.md

```diff
@@ -15,3 +15,12 @@ I am expecting (hopefully) to not be the only one to contribute to this project
 * Get information of an application
 * Displaying Weather on a watchface
 * Much much much more
+
```

Comment:
In the TODO,
 Replying to Pebble’s first request (PhoneVersion) [Doesn’t seem to work yet…], it could be set to completed :)